persistence/src/test/java/com/epam/rft/atsy/persistence/repositories/CandidateRepositoryIT.java

```diff
@@ -126,6 +138,47 @@ private void assertCandidateList(List<CandidateEntity> expectedCandidates,
   }
 
   @Test
+  public void findByCandidateFilterRequestShouldFindAllCandidatesWithEmptyFilterRequest() {
+    //Given
+    String name = "";
+    String email = "";
+    String phone = "";
+    String position = "";
+    List<CandidateEntity> expectedCandidates = Arrays.asList(candidateA, candidateB, candidateC);
+
+    //When
+    List<CandidateEntity>
+        actualCandidate =
+        candidateRepository
+            .findByCandidateFilterRequest(name, email, phone, position, DEFAULT_PAGEREQUEST)
+            .getContent();
+
+    //The
+    assertCandidateList(expectedCandidates, actualCandidate);
+  }
+
+  @Test
+  public void findByCandidateFieldRequestShouldFindCandidateCOnlyByPositionFilter() {
+    //Given
+    String name = "";
+    String email = "";
+    String phone = "";
+    String position = "f";
```

Comment:
Please refactor "f" string literal to a constant. Could you please do it with other string literals in this test class?